docs/src/modules/components/AppFrame.js

```diff
@@ -11,7 +13,7 @@ import Tooltip from '@material-ui/core/Tooltip';
 import NoSsr from '@material-ui/core/NoSsr';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import MenuIcon from '@material-ui/icons/Menu';
-import LanguageIcon from '@material-ui/icons/Language';
+import LanguageIcon from '@material-ui/icons/Translate';
```

Comment:
Use the same icon as React https://reactjs.org/
![Capture d’écran 2019-08-02 à 18 25 53](https://user-images.githubusercontent.com/3165635/62384572-05755780-b553-11e9-9cbb-140667e0d6f1.png)